packages/server/dataloader/azureDevOpsLoaders.ts

```diff
@@ -481,6 +506,40 @@ export const azureDevOpsWorkItem = (
   )
 }
 
+export const getMappedAzureDevOpsWorkItem = async (
+  manager: AzureDevOpsServerManager,
+  userId: string,
+  teamId: string,
+  instanceId: string,
+  returnedWorkItem: WorkItem
+) => {
+  const azureDevOpsWorkItem = {
+    id: returnedWorkItem.id.toString(),
+    title: returnedWorkItem.fields['System.Title'],
+    teamProject: getProjectId(new URL(returnedWorkItem.url)),
+    url: returnedWorkItem.url,
+    state: returnedWorkItem.fields['System.State'],
+    type: returnedWorkItem.fields['System.WorkItemType'],
+    descriptionHTML: returnedWorkItem.fields['System.Description']
+      ? returnedWorkItem.fields['System.Description']
+      : '',
+    service: 'azureDevOps'
+  } as AzureDevOpsWorkItem
+
+  const projectResult = await manager.getProjectProcessTemplate(
+    instanceId,
+    azureDevOpsWorkItem.teamProject
+  )
+  const {error: projectResultError, projectTemplate} = projectResult
+  if (!!projectResultError) {
+    const workItemId = returnedWorkItem.id.toString()
+    sendToSentry(projectResultError, {userId, tags: {instanceId, workItemId, teamId}})
+  } else {
+    azureDevOpsWorkItem.type = `${projectTemplate}:${returnedWorkItem.fields['System.WorkItemType']}`
+  }
+  return azureDevOpsWorkItem
+}
+
 export const azureDevOpsWorkItems = (
```

Comment:
Is this dataLoader being used anywhere?

Reply:
This was added to provide DataLoader capability to around just the getWorkItemData API call in case you wanted to enhance the functionality later.  Right now that API call is being called from the graphql schema under estimates at the individual work item, or from an orchestration cal

Reply:
If you would like it removed, just let me know.